winui/DataGrid/Export-To-Excel.md

```diff
@@ -0,0 +1,653 @@
+options.ExportMode = ExportMode.Text;
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample"); 
+{% endhighlight %}
+{% endtabs %}
+
+### Export groups with outlines
+
+By default, all the groups in dataGrid will be exported in expanded state. You can enable outlines in excel based on group expanded state by setting the `ShowOutlines` property as `true` in `DataGridExcelExportOptions`.
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+options.ShowOutlines = true;
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-to-Excel_images/winui-datagrid-export-to-excel.png" alt="Exporting to Excel with Outlines for Groups in WinUI DataGrid" width="100%" Height="Auto"/>
+
+### Exclude columns while exporting
+
+By default, all the columns (including hidden columns) in SfDataGrid will be exported to Excel. If you want to exclude some columns while exporting to Excel, you can use `ExcludedColumns` field in `DataGridExcelExportOptions`.
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+options.ExcludedColumns.Add("CustomerName");
+options.ExcludedColumns.Add("Country");
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+Here, the columns having `CustomerName` and `Country` as MappingName are excluded while exporting.
+
+### Excel version
+
+While exporting to Excel, you can specify the excel version by using `ExcelVersion` property.
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+### Exporting stacked headers
+
+You can export stacked headers to excel by setting `CanExportStackedHeaders` property to `true`.
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+options.CanExportStackedHeaders = true;
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+### Exporting merged cells
+
+You can export merged cells to excel by setting `CanExportMergedCells` property as `true`.
```

Comment:
Export the merged cells to excel by setting the `CanExportMergedCells` property as `true.`

Reply:
Content modified